research/object_detection/builders/anchor_generator_builder.py

```diff
@@ -78,5 +79,15 @@ def build(anchor_generator_config):
         anchor_offsets=anchor_offsets,
         reduce_boxes_in_lowest_layer=(
             ssd_anchor_generator_config.reduce_boxes_in_lowest_layer))
+  elif anchor_generator_config.WhichOneof(
```

Comment:
We should remove elif here and just use if since the cases above early return. This is more consistent with the rest of the code base.

Reply:
Hey Derek, I think elif is useful for getting into the final raise ValueError if non of the cases are satisfied.

Reply:
Ack, sorry for the double comment.

You can still use if. Looking at the statements above, if any of the above statements are true, the function will early return. Instead of the following pattern:

if:
  return
elif:
  return
... # More elifs
else:
  raise ValueError

In most other builders, we use the following style:

if:
  return
if
  return
# drop the final else statement
raise ValueError

We can do this because all the preceding if statements will early return.

Reply:
It's not needed. Currently the style here looks like this:

if:
  return
elif:
  return
# more elifs
else
  raise ValueError

Because each of the cases early returns, we can drop the elif and use if instead. This is more consistent with the other builders:

if:
  return
# repeats
raise ValueError